cmsplugin_feed_ai/cms_plugins.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+from django.utils.translation import ugettext_lazy as _
+from facebook import GraphAPIError
```

Comment:
Feels bad that the CMS plugin should have to know about the underlying API libraries.

Maybe add `.providers.excs`, and create a `FeedException` class that wraps these:

```python
class FeedException(Exception):
    def __init__(self, exc):
        super().__init__('%s: %s' % (exc.__class__, exc))
        self.original_exception = exc
```

and then catch only those here? :)